tests/neg-custom-args/fatal-warnings/i15503a.scala

```diff
@@ -68,7 +68,7 @@ object InlineChecks:
     inline def getSet = Set(1)
 
   object InlinedBar:
-    import collection.mutable.Set // ok
+    import collection.mutable.Set // error
```

Comment:
@szymon-rd 
I was just wondering about this one, it seems like this import is not required. It can be removed and still get the correct result : https://scastie.scala-lang.org/9pyvzVWyTAyBsOPduePhqg
Am I missing something ?

Reply:
No, you are correct. Some of these tests lay out the expected false negatives. If this is not a false negative, then great.